src/main/kotlin/kotliquery/QueryExtension.kt

```diff
@@ -0,0 +1,50 @@
+package kotliquery
+
+import kotliquery.action.ResultQueryActionBuilder
+import java.sql.Types
+
+fun Query.mapper() : ResultQueryActionBuilder<MutableMap<Any, Any?>> {
+    return ResultQueryActionBuilder(this, ::extractor)
+}
+
+fun extractor(row: Row): MutableMap<Any, Any?> {
```

Comment:
If you define this function as `toMap` and don't bring any extension methods for this purpose, I'm happy to merge this.